File: giliran_ku/backend/models/patient.go
package models

type Patient struct {
	Nik        string `json:"nik"`
	Name       string `json:"name"`
	Phone      string `json:"phone,omitempty"`
	Email      string `json:"email,omitempty"`
	BpjsNumber string `json:"bpjs_number,omitempty"`
}

File: giliran_ku/lib/data/models/patient.dart
class Patient {
  final String nik;
  final String name;

  const Patient({
    required this.nik,
    required this.name,
  });

  factory Patient.fromJson(Map<String, dynamic> json) {
    final name = json['name'] ?? json['patient_name'] ?? json['patientName'];
    return Patient(
      nik: json['nik'] as String,
      name: name as String,
    );
  }
}
